README.md

```diff
@@ -291,5 +293,12 @@ function toInt (v) {
 app.use(mw.body('key1, key2').require().validate(hasLengthOf3).transform(toInt));
 ```
 
-# License
-### MIT
+## License
+**MIT**
+
+[1]: https://travis-ci.org/tjmehta/dat-middleware.png
+[2]: https://travis-ci.org/tjmehta/dat-middleware
+[3]: https://david-dm.org/tjmehta/dat-middleware.png
+[4]: https://david-dm.org/tjmehta/dat-middleware
+[5]: https://david-dm.org/tjmehta/dat-middleware/dev-status.png
+[6]: https://david-dm.org/tjmehta/dat-middleware#info=devDependencies
```

Comment:
can you replace `[1-6]` with names to make them slightly more descriptive? should make it less of a mystery when one is looking at the badge line and wondering what the numbers are w/o going back and forth :)